tests/core/test_dataset_filters.py

```diff
@@ -3947,12 +3947,12 @@ def test_bounding_box_return_meta(oriented, as_composite):
     obb, point, axes = box_mesh.bounding_box(
         oriented=oriented, return_meta=True, as_composite=as_composite
     )
-
+    ATOL = 1e-6
```

Comment:
Please add a comment as to why you set this value. I think it is a numerical error, but I would like to clarify why it is not the default value.

Reply:
Add in https://github.com/pyvista/pyvista/pull/6524/commits/831c555fb4eafaaa958376b038863d9adce95557